docs/installLiberty.md

```diff
@@ -57,47 +57,36 @@ Use the `install` to specify the name of the Liberty server to install from the
 
 #### Examples
 
-1. Install using Liberty repository.
+1. Install Liberty runtime with all Java EE 7 features using Liberty repository.
   ```groovy
     apply plugin: 'liberty'
 
     liberty {
         install {
-            licenseCode = "<license code>"
-        }
-    }
-  ```
-
-2. Install from a specific location.
-  ```groovy
-    apply plugin: 'liberty'
-
-    liberty {
-        install {
-            licenseCode = "<license code>"
-            runtimeUrl = "<url to runtime.jar>"
+            type = "javaee7"
         }
     }
   ```
 
-3. Install Liberty runtime with all Java EE 7 features using Liberty repository.
+2. Install from a specific location using a zip file.
   ```groovy
     apply plugin: 'liberty'
 
     liberty {
         install {
-            type = "javaee7"
+            runtimeUrl="<url to wlp*.zip>"
         }
     }
   ```
-
-4. Install from a specific location using a zip file.
+  
+3. Install from a specific location.
```

Comment:
3. Install from a specific location using a jar file. `licenseCode` is required when installing from a jar file.